<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterAdjustmentItem</class>
 <widget class="QWidget" name="ParameterAdjustmentItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>前缀：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_le_PrefixBytes"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>后缀：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_le_SuffixBytes"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item alignment="Qt::AlignLeft">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>格式：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_cb_Format">
         <item>
          <property name="text">
           <string>int16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>float</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>缩放：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_le_Scale">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
